cyhy_report/customer/report.mustache

```diff
@@ -609,7 +611,7 @@ As you review the report, you may have additional questions. Check out the answe
 
         % blank line needed on previous line for vertical spacing
         \numprint{<<&calc.addresses_scanned_percent>>}\% of addresses scanned}
```

Comment:
@jeffkause Did you also want this wording change in this PR?
```suggestion
        \numprint{<<&calc.addresses_scanned_percent>>}\% of assets scanned}
```

Reply:
@jeffkause Following up on this last remaining open conversation in this PR - please see my comment above and let us know if you want to accept my suggestion.

Reply:
Hi @dav3r @jeffkause , for the file cyhy_report/customer/report.mustache line 613, dav3r raised the question whether the line should use the wording 'assets' versus 'addresses'. The current version of the file reads as 'addresses'. Has this discussion been resolved?